Core/include/Acts/EventData/VectorMultiTrajectory.hpp

```diff
@@ -307,10 +333,12 @@ class VectorMultiTrajectory final
   std::vector<IndexType> m_previous;
   std::vector<typename detail_lt::Types<eBoundSize>::Coefficients> m_params;
   std::vector<typename detail_lt::Types<eBoundSize>::Covariance> m_cov;
-  std::vector<typename detail_lt::Types<MeasurementSizeMax>::Coefficients>
-      m_meas;
-  std::vector<typename detail_lt::Types<MeasurementSizeMax>::Covariance>
-      m_measCov;
+
+  std::vector<double> m_meas;
```

Comment:
So this is now a flat vector which can only be used with offsets to get the measurements/covariances?

Reply:
In this implementation: yes. My thinking was that on the xAOD side it's `std::vector<std::vector<double>>` anyway, so the access remains trivial.